libyul/backends/wasm/WasmDialect.cpp

```diff
@@ -98,6 +98,8 @@ WasmDialect::WasmDialect()
 	// We could introduce "i32.drop".
 	addFunction("drop", "i64"_yulstring, 1, 0);
 
+	addFunction("nop", "i64"_yulstring, 0, 0, true, false);
```

Comment:
Nop takes no argument, so `i64_yulstring` is misleading.

Reply:
Not misleading. Wrong.

Reply:
Well isn't `0, 0` meaning 0 argument and 0 results?

Reply:
Ah sorry, now I understand.